.github/ISSUE_TEMPLATE/new-dbt-model.md

```diff
@@ -157,6 +195,53 @@ using (pin10, year)
      data_tests:
 ```
 
+- [ ] _[Python models only]_ If you need any third-party pure Python packages
```

Comment:
**suggestion (non-blocking):** I would maybe just add a note here linking README section that details the format used to specify dependencies.

Reply:
Good call, done in [a8ed694](https://github.com/ccao-data/data-architecture/commit/a8ed694464bf18cb0df1b412b08f696b3fb5a3f5).